src/ILVerify/src/ILImporter.Verify.cs

```diff
@@ -1694,6 +1733,83 @@ void CheckPendingPrefix(Prefix mask)
             Check((mask & Prefix.Constrained) == 0, VerifierError.Constrained);
         }
 
+        bool HasPendingPrefix(Prefix prefix)
+        {
+            return (_pendingPrefix & prefix) != 0;
+        }
+
+        void ClearPendingPrefix(Prefix prefix)
+        {
+            _pendingPrefix &= ~prefix;
+        }
+
+        void VerifyPendingPrefix(ILOpcode opCode)
+        {
+            if (_pendingPrefix == 0)
+                return;
+
```

Comment:
Does this handle multiple prefixes well? (E.g. `unaligned.` and `volatile` can be combined together.)